### PR TITLE
feat: event card

### PR DESCRIPTION
**Pull request by in-mai-space**
# Description

[Link to Ticket](https://trello.com/c/BbtI2mVp/22-mobile-event-card-component-fe)
Simulator Screenshot - iPhone 15 Pro Max - 2024-05-30 at 22 10 12
Simulator Screenshot - iPhone 15 Pro Max - 2024-05-30 at 22 10 53

# How Has This Been Tested?

No tests.

# Checklist

- [x] I have performed a self-review of my code
- [x] I have reached out to another developer to review my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by alderwhiteford — APPROVED** [review]

**Review by garrettladley — APPROVED** [review]
LGTM @in-mai-space 👑 once the merge conflicts are resolved, we're good to go

**Pull request merged**
